### PR TITLE
Display localized car names in debug menu

### PR DESCRIPTION
**Pull request by BrettDong**
and a trivial fix prevents 'none' from appearing in the vehicle list.

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -4152,11 +4152,11 @@ void game::debug()
             } else {
                 std::vector<vproto_id> veh_strings;
                 for( auto & elem : vehicle_prototype::get_all() ) {
-                    if( elem != vproto_id( "custom" ) ) {
+                    if( elem != vproto_id( "custom" ) && elem != vproto_id( "none" ) ) {
```
Comment (mugling) [inline src/game.cpp]: This change doesn't look right and should be handled in a separate PR

Comment (BrettDong) [inline src/game.cpp]: Fine, I'm splitting it to another PR.

Comment (mugling) [inline src/game.cpp]: We need to find out why it requires a 'magic type' first
